docs/GRAPHQL.md

```diff
@@ -5,20 +5,23 @@
 ### `[GET|POST] /api/v2.2/graphql`
 
 Accepts `GET` or `POST` requests to perform GraphQL queries. See:
-- https://www.graphql-java.com/tutorials/getting-started-with-spring-boot/
-- https://graphql.org/learn/serving-over-http/
+- [https://www.graphql-java.com/tutorials/getting-started-with-spring-boot/](Getting Started)
+- [https://graphql.org/learn/serving-over-http/](Serving over HTTP)
 for some more info.
 
-src/main/java/io/cryostat/net/web/http/api/v2/graph/GraphModule.java
-contains the bindings for the GraphQL engine to query custom targets.
-
-`graphql/` contains some sample queries which can be used as in the following
-example queries. The first query is a standard API v2 requst to create a custom
+[`graphql/`](graphql/) contains some sample queries which can be used as in the following
+example queries. The first query is a standard API v2 request to create a custom
 target. The second query is a standard API v1 request to list all known targets.
 The third query is a GraphQL query listing all known targets and all fields of
 those targets, except without querying for the recordings belonging to the
 target.
 
+For more information about the available queries and mutations check the
+GraphiQL interface detailed below, or check the GraphQL schema definitions:
```

Comment:
typo at `GraphiQL`?

Reply:
Probably we should add a comma at `...mutations, check...`

Reply:
No, [`GraphiQL`](https://github.com/graphql/graphiql) is intentional. It's the name of the "IDE" view.